src/project/FileTreeView.js

```diff
@@ -59,6 +59,8 @@ define(function (require, exports, module) {
     var CLICK_RENAME_MINIMUM  = 500,
         RIGHT_MOUSE_BUTTON    = 2,
         LEFT_MOUSE_BUTTON     = 0;
+    
+    var INDENTATION_WIDTH     = 18;
```

Comment:
Should this be configurable? It wasn't with the CSS but could be a nice to have when it's in code.